README.rst

```diff
@@ -14,23 +14,103 @@ Lasagne
 =======
 
 Lasagne is a lightweight library to build and train neural networks in Theano.
+Its main features are:
 
-For support, please refer to the lasagne-users mailing list: https://groups.google.com/forum/#!forum/lasagne-users
+* Support for feed-forward networks including Multi-Layer Perceptrons (MLPs)
+  and Convolutional Neural Networks (CNNs)
+* Support for recurrent networks including Long Short-Term Memory (LSTM) and
+  Gated Recurrent Units (GRUs) [coming soon!]
+* Support for architectures of multiple inputs and multiple outputs, including
+  auxiliary classifiers
+* Support for many optimization methods including Nesterov momentum, RMSprop
+  and ADAM
```

Comment:
That's right, but it felt weird to list "feed-forward networks" and "architectures" as features. We need a different introduction for the list then, one which fits "Feed-forward networks", "Optimization methods", but also "Designed to be easy to use and easy to extend". I'll figure something out.